FireRender.Maya.Src/pluginMain.cpp

```diff
@@ -559,7 +559,8 @@ MStatus initializePlugin(MObject obj)
 	pTopLevelExceptionFilter = ::SetUnhandledExceptionFilter(FrUnhandledExceptionFilter);
 #endif
 
-	PluginUpdater();
+	// We have legacy updater here which does not work. Comment this code for now becaue it breaks Maya 2022 startup.
+	//PluginUpdater();
```

Comment:
Keeping this code commented out is intentional, right?
So that we don't forget to fix this feature in the future?

Reply:
Yeah I would prefer not to remove it right now. Not sure we gonna make this feature alive though, but let's keep for now